pkg/apis/apiserver/organization_REST.go

```diff
@@ -350,7 +360,45 @@ func internalErrorWatchEvent(err error) watch.Event {
 	}
 }
 
-func (o *OrganizationREST) isVisible(ctx context.Context, organization *Organization) (bool, error) {
+func (o *OrganizationREST) checkIsOwner(ctx context.Context, organization *Organization) error {
+	attrs, err := filters.GetAuthorizerAttributes(ctx)
+	if err != nil {
+		return err
+	}
+	visible, err := o.isMember(ctx, organization)
```

Comment:
This is not needed, you don't need to check if Owner is member.

Reply:
Yes, it's needed. It's a different error message. The difference between `NotFound` and `Forbidden` should be honored.

Reply:
Ok, thanks, this makes sense